configs/text-gen-service/gce.yaml

```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head_node_type
+  instance_type: n1-standard-4-nvidia-t4-16gb-1
+worker_node_types:
+- name: gpu_worker
+  instance_type: n1-standard-4-nvidia-t4-16gb-1
+  min_workers: 1
+  max_workers: 3
```

Comment:
Should this be the same `max_workers` as in `aws`?

Reply:
fixed

Reply:
Done